src/openforms/conf/base.py

```diff
@@ -923,7 +923,7 @@
 and it plays nice with other available components.
 """
 
-API_VERSION = "2.6.0-alpha.0"
+API_VERSION = "2.6.0"
```

Comment:
Same question about about the missing `-beta`.
Maybe it's because this is about the public endpoints and those won't change between now and the release?

Reply:
Same as above.